.github/workflows/test-suite.yml

```diff
@@ -400,3 +400,40 @@ jobs:
       run: rustup override set beta
     - name: Run make
       run: make
+  # This job succeeds ONLY IF all others succeed. It is used by the merge queue to determine whether
+  # a PR is safe to merge. New jobs should be added here.
+  test-suite-success:
+    name: test-suite-success
+    runs-on: ubuntu-latest
+    needs: [
+        'target-branch-check',
+        'extract-msrv',
+        'cargo-fmt',
+        'release-tests-ubuntu',
+        'release-tests-windows',
+        'beacon-chain-tests',
+        'op-pool-tests',
+        'slasher-tests',
+        'debug-tests-ubuntu',
+        'state-transition-vectors-ubuntu',
+        'ef-tests-ubuntu',
+        'dockerfile-ubuntu',
+        'eth1-simulator-ubuntu',
+        'merge-transition-ubuntu',
+        'no-eth1-simulator-ubuntu',
+        'syncing-simulator-ubuntu',
+        'doppelganger-protection-test',
+        'execution-engine-integration-ubuntu',
+        'check-benchmarks',
+        'clippy',
+        'check-msrv',
+        'arbitrary-check',
+        'cargo-audit',
+        'cargo-vendor',
+        'cargo-udeps',
+        'compile-with-beta-compiler'
+    ]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Check that success job is dependent on all others
+      run: ./scripts/ci/check-success-job.sh ./.github/workflows/test-suite.yml test-suite-success
```

Comment:
This script could possibly be reused for `linkcheck` and `local-testnet`, although I noticed they only had 1 job each, with a matrix in the case of `local-testnet`. Given the YAML introspection approach, I'm not sure it's worth defining a similar success job for local-testnet. Maybe we can just add it in the UI? (possibly with the OS variants from the `matrix`).

Reply:
Nvm, seems it was easy